dig.go

```diff
@@ -258,6 +262,13 @@ func setRand(r *rand.Rand) Option {
 	})
 }
 
+// Dry sets container mode to dry used for testing graphs without invocation.
```

Comment:
Since this is the exported API, I would make this comment a little more detailed. Maybe something like:
```
// Dry provides a "dry run" testing option that, if set to true, will
// verify the graph without invocation. 
```

Reply:
Fair, comment is a bit clumsy.